woocommerce/woocommerce
# Fixed WooCommerce menu for users that can only manage orders on WooCommerce

### All Submissions:






### Changes proposed in this Pull Request:



If you remove `manage_woocommerce` capability from a Shop Manager the WooCommerce menu starts to lost its items, see:

![Screenshot from 2020-06-25 20-24-06](https://user-images.githubusercontent.com/1264099/85805154-e90d7b00-b721-11ea-8b13-9aaa1d4c11dc.png)
 
Note that the "WooCommerce" menu only shows now "Customers" and "Dashboard".

Removing the `manage_woocommerce` or creating an user to only manage orders is a common thing to avoid high privileged users managing your store.

Closes #26547.

### How to test the changes in this Pull Request:

1. Create a shop manager user on your installation.
2. Remove the `manage_woocommerce` cap.
```php
global $wp_roles;
$wp_roles->remove_cap( 'shop_manager', 'manage_woocommerce' );
```
3. Check that the WooCommerce menu is all over the place now.
4. Apply this patch, it should display like:

![Screenshot from 2020-06-25 19-59-40](https://user-images.githubusercontent.com/1264099/85804945-5c62bd00-b721-11ea-90a3-914fbd05e306.png)

Notes: There's some menu items that need to get fixed on WooCommerce Admin side, there's why there's an empty space in the WooCommerce menu, and that's why Analytics and Marketing are also not included.
cc @woocommerce/isotope 

### Other information:




### Changelog entry

> Tweak - Fixed WooCommerce menu for users that can only manage orders on WooCommerce.
